src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexFCD.cs

```diff
@@ -400,27 +359,27 @@ private void CalculateFC(int NodeType, RegexNode node, int CurIndex)
                     break;
 
                 case RegexNode.Empty:
-                    PushFC(new RegexFC(true));
+                    _fcStack.Add(new RegexFC(true));
                     break;
 
                 case RegexNode.Concatenate | AfterChild:
                     if (CurIndex != 0)
                     {
-                        RegexFC child = PopFC();
-                        RegexFC cumul = TopFC();
+                        RegexFC child = FCPop();
+                        RegexFC cumul = _fcStack[_fcStack.Count - 1];
```

Comment:
Why did you change all of these away from the helper method?

Reply:
When I was trying a different data model I probably removed those. When I prepared the PR today for review today I noticed that I should revert these changes but was too lazy. Will do that now ;)